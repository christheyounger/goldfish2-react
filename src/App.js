import React, { Component } from "react";
import "./App.css";
import Header from "./component/Header";
import TasksList from "./screens/TasksList";
import "bootstrap/dist/css/bootstrap.css";

class App extends Component {
  render() {
    return (
      <div className="App">
        <Header />
        <TasksList />
      </div>
    );
  }
}

export default App;
