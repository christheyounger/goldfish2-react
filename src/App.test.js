import React from "react";
import render from "react-test-renderer";
import App from "./App";

it("renders without crashing", () => {
  const wrapper = render.create(<App />);
  expect(wrapper.toJSON()).toMatchSnapshot();
});
