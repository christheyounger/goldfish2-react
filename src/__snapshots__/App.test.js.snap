// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <nav
    className="navbar navbar-light bg-light"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        alt=""
        src="goldfish.svg"
        style={
          Object {
            "height": 30,
          }
        }
      />
      Goldfish
    </a>
  </nav>
  <div>
    <h2>
      Today's Tasks:
    </h2>
    <table
      className="table"
    >
      <thead>
        <tr>
          <th />
          <th>
            Task
          </th>
          <th>
            Due
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
    <button
      onClick={[Function]}
    >
      Hello
    </button>
  </div>
</div>
`;
